### ISSUE TITLE
Date Taken Option

### ISSUE THREAD

**Issue by George-1958** [diffractor/diffractor#106]
At the moment it seems to display and change the Date Created. Can you please have an option to Display and Change the Date Taken instead ?  thanks

**Comment by ZacWalk** [comment]
Yes - that is the case. A data change does cause both the metadata date to be updated and also the file created date.
It is a good idea to have an option.

**Comment by ZacWalk** [comment]
Closed in the latest version (v1.26.3). Please re-open if you still see any issues. Thanks!
